ssg.js

```diff
@@ -39,16 +43,22 @@ if (config.parsedParams.help) {
  * At this point we know that we will have do do the work
  */
 
-// Verify input is given
-if (!config.parsedParams.input) {
-  // eslint-disable-next-line no-console
-  console.log('Input parameter missing');
+// Verify input or config is given
+let outputDirPath = DEFAULT_RESULT_FOLDER;
+let language = 'en-CA';
+if (config.parsedParams.input) {
+  language = config.parsedParams.language || language;
+  outputDirPath = config.parsedParams.output || outputDirPath;
+} else if (config.parsedParams.config) {
+  config = parseConfigArgs(config.parsedParams.config);
+  language = config.parsedParams.language || language;
+  outputDirPath = config.parsedParams.output || outputDirPath;
+} else {
+  // eslint-disable-next-line
+  console.log('Input or config parameter missing');
   process.exit(1);
 }
 
-const { language = 'en-CA', output: outputDirPath = DEFAULT_RESULT_FOLDER } =
-  config.parsedParams;
-
```

Comment:
Your solution would require us fo modify this merging code for every new config param introduced. Please use a solution like this instead

```suggestion
let { parsedParams } = config;
if (parsedParams.config) {
  parsedParams = {
    ...parsedParams,
    ...parseConfigArgs(config.parsedParams.config).parsedParams
  };
}
/ Verify input is given
if (!parsedParams.input) {
  // eslint-disable-next-line no-console
  console.log('Input parameter missing');
  process.exit(1);
}
const { language = 'en-CA', output: outputDirPath = DEFAULT_RESULT_FOLDER } =
  parsedParams;
```

Reply:
Hi Denes, the following recomendation file is failing when I run with -config. I am getting error:  
No txt files to process

Reply:
If you also change 
`inputPath: config.parsedParams.input,` to `inputPath: parsedParams.input,`
in line `67`, it should work (I tested it)

Reply:
I fixed it. Please approve the PR and merge.